src/server/scripts/Northrend/IsleOfConquest/isle_of_conquest.cpp

```diff
@@ -31,50 +31,39 @@
 // TO-DO: This should be done with SmartAI, but yet it does not correctly support vehicles's AIs.
```

Comment:
Cosmetic/grammar: The location of "yet" affects its effective meaning in a sentence.
```suggestion
// TO-DO: This should be done with SmartAI, but it does not yet correctly support vehicles' AIs.
```
Also: Remove redundant possessive 'S' which is represented by the apostrophe after plural nouns ending with 'S'.

Reply:
Not for this PR, even such small changes